delta/plugins/composite-views/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/CompositeViewDecodingSpec.scala

```diff
@@ -111,6 +111,7 @@ class CompositeViewDecodingSpec
       ElasticSearchProjectionFields(
         Some(iri"http://music.com/bands"),
         query1,
+        Some("cv"),
```

Comment:
We should also put a length limit and sanitization (maybe use a label here?) to this along with tests to avoid very long index names.